requirements.txt

```diff
@@ -8,3 +8,4 @@ PyYAML
 koji
 psutil
 moksha.hub
```

Comment:
You probably don't need this one anymore.

Reply:
Probably.